index.js

```diff
@@ -37,7 +37,10 @@ module.exports = function svgLoader (options = {}) {
       }
 
       if (svgo !== false) {
-        svg = optimizeSvg(svg, svgoConfig).data
+        svg = optimize(svg, {
+          ...svgoConfig,
+          path,
```

Comment:
Maybe better to turn these around? To make it possible to set the path in the `svgoConfig`